recipes/newrelic/infrastructure/logs/linux-logs.yml

```
@@ -81,6 +81,11 @@ installTargets:
     platformFamily: rhel
     platformVersion: "((7|8)\\.?.*)"
     kernelArch: aarch64
+  - type: host
+    os: linux
+    platform: amazon
+    platformVersion: "2023"
```

The platform version for the AMIs don't match, please change it to 
```suggestion
    platformVersion: "(2023\\.*)"
```